cloudify_azure/connection.py

```diff
@@ -92,10 +92,13 @@ def request(self, **kwargs):
         if url.startswith('/'):
             # Add the endpoint and subscription ID
             url = '{0}/subscriptions/{1}{2}'.format(
-                constants.CONN_API_ENDPOINT,
+                creds.endpoints_resource_manager,
```

Comment:
Is this Azure Stack specific?

Reply:
Yes, AzureStack can use different url for resource managment.